src/Account/Controller/UserRelationshipController.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  forwardRef,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Inject,
+  Param,
+  ParseIntPipe,
+  Put,
+  Query,
+} from '@nestjs/common';
+import {
+  ApiBadRequestResponse,
+  ApiCreatedResponse,
+  ApiNoContentResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+} from '@nestjs/swagger';
+import UserFollowService from '../Service/UserFollowService';
+import ObjectIdValidationPipe from '../../Common/Pipe/ObjectIdValidationPipe';
+import SwaggerBadRequestResponse from '../../Common/SwaggerBadRequestResponse';
+import { ConfigService } from '@nestjs/config';
+import BusinessError from '../../Common/BusinessError';
+import { ErrorCode } from '../../ErrorCode';
+import UserService from '../Service/UserService';
+import { GetFollowListResponse } from './Response/UserRelationshipResponse';
+import { CreateFollowDto } from './Dto/UserRelationshipControllerDto';
+
+@ApiTags('User Service')
+@Controller('/user')
+export default class UserRelationshipController {
+  @Inject()
+  private userFollowService: UserFollowService;
+
+  @Inject()
+  private configService: ConfigService;
+
+  @Inject(forwardRef(() => UserService))
```

Comment:
Please correct me if I'm wrong, I didn't see any Circular dependency here between UserRelationshipController and UserService that requires `forwardRef`.

Reply:
You are right,  This is what I copied from other files and forgot to modify

Reply:
[(Fix and e2e test) 3aa2739](https://github.com/haozi23333/test-backend/commit/3aa27391dd3bab130d86dfd0751a0860e7cbeb91)